testing/api.spec.js

```diff
@@ -64,6 +62,21 @@ describe('API calls', () => {
       .then(resp => expect(resp.data).toEqual(overview));
   });
 
+  test('/overview/pricing should fetch object with expected properties per app and service plan', async () => {
```

Comment:
Great job pairing a test with your update! I'm impressed and need to start doing this as well.